python/cudf/.coveragerc

```diff
@@ -1,4 +1,4 @@
 # Configuration file for Python coverage tests
 [run]
-include = cudf/*
-omit = cudf/tests/*
\ No newline at end of file
+source = cudf
+omit = cudf/tests/*
```

Comment:
Do we really want to omit our tests? Unused functions in our tests should be flagged as "not covered."